pkg/k8s/resource/resource.go

```diff
@@ -180,6 +192,36 @@ func fetchKubeSystemRoleBindings(ctx context.Context, k *k8s.KubernetesAPI, opti
 	return resources, nil
 }
 
+func fetchControlNamespaceRoles(ctx context.Context, k *k8s.KubernetesAPI, options metav1.ListOptions, namespace string) ([]Kubernetes, error) {
+	list, err := k.RbacV1().Roles(namespace).List(ctx, options)
```

Comment:
Since we're using a labelSelector in the options to match only resources for a given extension we might be able to simplify the logic in this PR by getting roles from all namespaces; we can pass an empty string instead of a ns to do this.

```suggestion
	list, err := k.RbacV1().Roles("").List(ctx, options)
```

Reply:
Ooo, this is neat, didn't know about the empty string workaround. Since this does make things cleaner, I'll update my PR accordingly :)

Reply:
done! let me know if this is gtg, i'll squash my commits.

Reply:
wouldn't worry about squashing, it's a small change. Think whoever's going to merge this in will do it :)